drivers/base/bootvid/precomp.h

```diff
@@ -71,6 +77,18 @@ InitPaletteWithTable(
     _In_ PULONG Table,
     _In_ ULONG Count);
 
+VOID
+NTAPI
+VidpScroll(
+    _In_ ULONG Scroll);
+
+VOID
+NTAPI
+PreserveRow(
+    _In_ ULONG CurrentTop,
+    _In_ ULONG TopDelta,
+    _In_ BOOLEAN Restore);
+
 /*
```

Comment:
Can you move the globals just after the typedefs and before the macros/functions?